Library/Homebrew/extend/os/mac/diagnostic.rb

```diff
@@ -129,11 +129,17 @@ def check_for_unsupported_macos
         end
 
         def check_for_opencore
-          has_opencore =
-            File.exist?("/Library/PrivilegedHelperTools/com.dortania.opencore-legacy-patcher.privileged-helper")
-          has_opencore ||= File.exist?("/Library/Application Support/Dortania/OpenCore-Patcher.app")
-          has_opencore ||= MacOS.pkgutil_info("com.dortania.opencore-legacy-patcher").present?
-          return unless has_opencore
+          return if ::Hardware::CPU.physical_cpu_arm64?
+          return if ENV["CI"]
+
+          # https://dortania.github.io/OpenCore-Legacy-Patcher/UPDATE.html#checking-oclp-and-opencore-versions
+          begin
+            opencore_version = Utils.safe_popen_read("/usr/sbin/nvram",
+                                                     "4D1FDA02-38C7-4A6A-9CC6-4BCCA8B30102:opencore-version")
```

Comment:
```suggestion
                                                     "4D1FDA02-38C7-4A6A-9CC6-4BCCA8B30102:opencore-version").split[1]
```
Save just the version string.